### ISSUE TITLE
🐛 [BUG] - Deployment exposes port twice

### ISSUE THREAD

**Issue by IvoGoman** [cloudoperators/heureka#214]
### Priority

None

### Description

The Heureka Chart can currently not be installed, because the '.Values.service.port' is exposed twice as two different named ports. This is not allowed and results in: 
```
Helm diff failed: failed to server-side diff Deployment/heureka: failed
        to patch object: failed to create manager for existing fields: failed to convert
        new object (greenhouse/heureka; apps/v1, Kind=Deployment) to smd typed: .spec.template.spec.containers[name="heureka"].ports:
        duplicate entries for key [containerPort=8080,protocol="TCP"]
```

### Reproduction steps

```bash
1. Go to 'Greenhouse QA Dashboard'
2. View the Heureka Plugin's 'Raw Data'
3. Scroll down to '.status'
4. See error
```

**Comment by IvoGoman** [comment]
@drochow @dimtas could you take a look?

**Comment by IvoGoman** [comment]
The problem still persists: https://github.com/cloudoperators/heureka/blob/f04feabc6b81521f81efa368c0356b613e9fc763/charts/heureka/templates/deployment.yaml#L67-L72

**Comment by dimtas** [comment]
@IvoGoman can you please verify that this is fixed?

**Comment by IvoGoman** [comment]
Looks good with regards to the port. Could you provide an update to the [PluginDefinition](https://github.com/cloudoperators/greenhouse-extensions/blob/main/heureka/plugindefinition.yaml)?

Metrics scraping from within Greenhouse will not be possible with how the chart is currently configured, as it is hard-coded to the Kubernetes prometheis from our internal infrastructure.

**Comment by dimtas** [comment]
Currently we deploying the chart using helm manually and not using the gh workflow since it's in development. I guess this is why it doesn't shown.

**Comment by IvoGoman** [comment]
I only noticed this because the Chart is added to the PluginDefinition and thus gets deployed.  In this case the reconciliation for the chart is continuously failing.

**Comment by dimtas** [comment]
I see. So, if I bump up the version to 3.0.1 in here:
https://github.com/cloudoperators/greenhouse-extensions/blob/59d07c21da02176a5392511f3e4aacfaee8fd51f/heureka/plugindefinition.yaml#L18

This will suppress the warning?

**Comment by IvoGoman** [comment]
Yes, best to also bump the patch level on the [PluginDefinition](https://github.com/cloudoperators/greenhouse-extensions/blob/59d07c21da02176a5392511f3e4aacfaee8fd51f/heureka/plugindefinition.yaml#L10) as well

**Comment by IvoGoman** [comment]
I just noticed that the chart version was not bumped in this repo, but only in the `greenhouse-extensions`.

**Comment by dimtas** [comment]
Oh... I may missed that. I will fix this
